journal/modernized/2-journal.adoc

```diff
@@ -8175,14 +8223,16 @@ held at Mount Pleasant, the 16th of eleventh month, 1834.
 [.signed-section-signature]
 Benjamin W. Ladd, Clerk for the day.
 
-[.asterism]
-'''
+//^^ Not sure how "minute" should be labeled. I think there was another one above, and I can't remember if I made a note about it or not. Should it be embedded? I think I embedded the other one. 
```

Comment:
Ok, so I think we should embed this section, and use the new type `.minute`.  Start the embed right before the sentence that starts "Soon after this meeting directed the reprinting "